modos/storage.py

```diff
@@ -79,9 +79,8 @@ def list(self, target: Optional[Path] = None):
                     yield file
 
     def remove(self, target: Path):
-        path = self.path / target
-        if path.exists():
-            path.unlink()
+        if target.exists():
+            target.unlink()
```

Comment:
**question:** iiuc, this deletes by relative path instead of absolute, is this safe? Any particular reason to change it in this method specifically?

Reply:
Fair point! 
My reasoning was that `modo.list_files()` returns the absolute path already. By expecting `target` to be an absolute path, the code works conveniently for remote and local storage.  
In addition in https://github.com/sdsc-ordes/modos-api/blob/e0995de6e27fe6f649c78c82ca9e6f1f182f9d1a/modos/api.py#L186-L188 we parse the absolute path. This was not working with `storage.remove(target)` removing `path = self.path / target`, but there was a bug in `modo.remove_element()`. 
So this does fix the bug and allows using `self.list_files()`independent of local or remote storage.

Reply:
I added a test to check for file deletion in https://github.com/sdsc-ordes/modos-api/pull/76/commits/7790bd22764bb3478bc9c326fdc2a7f2156b05fc for resolving this :-)